src/flux/query_result/table.cr

```diff
@@ -5,7 +5,7 @@ struct Flux::QueryResult::Table(T)
 
   getter columns : Array(Column)
 
-  private getter records = [] of T
+  getter records = [] of T
```

Comment:
This is an internal data structure.

`Table(T)` is [`Indexable`](https://crystal-lang.org/api/0.36.1/Indexable.html) over the elements this holds.

Reply:
@GabFitzgerald